docs/source/adb.rst

```diff
@@ -0,0 +1,131 @@
+.. _adb:
+
+####################
+Android Debug Bridge
+####################
+
+.. important:: Android Debug Bridge is required ONLY for those users pairing an Android mobile device with Cozmo. iOS users require *usbmuxd*; instructions for installing this are given on the install pages for each operating system.
```

Comment:
Regarding 'iOS users require usbmuxd; instructions...are on the install pages for each OS'. That is a bit misleading because for Mac users, nothing has to be installed. Maybe adding 'if necessary' would clear it up a little.